bee-ledger/src/workers/snapshot/worker.rs

```diff
@@ -68,6 +69,7 @@ where
         // Unwrap is fine because snapshot info was either just inserted or already present in storage.
         let snapshot_info = storage::fetch_snapshot_info(&*storage)?.unwrap();
 
+        // TODO(Adam) extract to separate traced method.
```

Comment:
What does that mean ?

Reply:
It's something I left in there by accident -- I'll double check.